lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.scss

```diff
@@ -92,3 +103,28 @@
         }
     }
 }
+
+.app-metadata-tab .adf-metadata-properties .mat-expansion-panel {
```

Comment:
Same here, app-metadata-tab comes from ACA, not from ADF so should not be used here in ADF